### PR TITLE
Performance

### PR DESCRIPTION
**Pull request by koikiss-dev**

### CODE REVIEW

**Review by barrientosvctor — CHANGES_REQUESTED** [review]
I highly recommend to use `import type { }` / `import { type ... }` when importing a type instead of using `import {}`. This increases the app performance and it's only supported for TypeScript files.

- [Source](https://medium.com/@quizzesforyou/import-vs-import-type-in-typescript-8e5177b62bea)

File: src/scraper/sites/anime/animelatinohd/AnimeLatinoHD.ts [hunk 1]
```diff
@@ -2,104 +2,127 @@ import * as cheerio from "cheerio";
...
+      } else {
+        $("#languaje option").each((_i, e) => {
+          f_index = Number($(e).val());
+        });
+      }
+
+      await Promise.all(
+        animeEpisodeParseObj.players[f_index].map(
+          async (e: { server: { title: string }; id: string }) => {
+            //const min = await axios.get("https://filemoon.sx/e/smone1s7jjxv/CYM01HNMCGTSKT")
```
Comment (Mawfyy) [inline src/scraper/sites/anime/animelatinohd/AnimeLatinoHD.ts]: Please, don't put commented code, delete or do whatever you want with that code.

File: src/scraper/sites/anime/animelatinohd/AnimeLatinoHD.ts [hunk 2]
```diff
@@ -2,104 +2,127 @@ import * as cheerio from "cheerio";
...
+            //const pageload = await BrowserHandler("https://animelatinohd.com/")
+
+            const Server: EpisodeServer = {
+              name: e.server.title,
+              url: "",
+            };
+            //const cookies = [{name: 'v_id', value: "https://api.animelatinohd.com/stream/"+e.id},];
```
Comment (Mawfyy) [inline src/scraper/sites/anime/animelatinohd/AnimeLatinoHD.ts]: Please, don't put commented code, delete or do whatever you want with that code.

File: src/scraper/sites/anime/animelatinohd/AnimeLatinoHD.ts [hunk 3]
```diff
@@ -2,104 +2,127 @@ import * as cheerio from "cheerio";
...
+            Server.url = "https://api.animelatinohd.com/stream/" + e.id;
+            Server.name = e.server.title;
+
+            //await pageload.page.setCookie(...cookies)
```
Comment (Mawfyy) [inline src/scraper/sites/anime/animelatinohd/AnimeLatinoHD.ts]: Please, don't put commented code, delete or do whatever you want with that code.

File: src/scraper/sites/anime/gogoanime/Gogoanime.ts [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import { getHTML } from "./assets/getHTML";
+/* import { getHTML } from "./assets/getHTML";
```
Comment (Mawfyy) [inline src/scraper/sites/anime/gogoanime/Gogoanime.ts:1]: Please, don't put commented code, delete or do whatever you want with that code.

**Review by Mawfyy — COMMENTED** [review]
I like this change, more modular but, i prefer quit commented code from branch like dev or main.

**Review by barrientosvctor — APPROVED** [review]
It looks great. Good job!

**Comment by barrientosvctor** [comment]
btw, remember to resolve the conflicts on `Monoschinos.ts`. The most recent change was in dev branch.

**Pull request merged**